RELEASING-AMAZON.md

```diff
@@ -0,0 +1,11 @@
+1. Start a branch `release/amazon-x.x.x`
+1. Run `flutter format .`
+1. Update CHANGELOG-LATEST.md with the changes for the current version (to be used by Fastlane for the github release notes)
+1. Run `bundle exec fastlane bump_and_update_changelog version:X.Y.Z` (where X.Y.Z is the new version) to update the version number in pubspec.yaml, purchases_flutter.podspec and android/build.gradle.
+1. `flutter pub pub publish --dry-run`
+1. If purchases-hybrid-common was updated, run `pod update PurchasesHybridCommon` in both `MagicWeather` and `purchase_tester`
+1. Commit the changes, open a PR into main, merge when ready
```

Comment:
😄

Reply:
I'm going to start a copy / paste jar where I have to put a dollar in whenever I make a copy / paste bug. 
In a year I'll use it to buy a car.